components/inputs/input-date-time.js

```diff
@@ -43,6 +43,10 @@ class InputDateTime extends FormElementMixin(LocalizeCoreElement(RtlMixin(LitEle
 			 * Indicates that a value is required
 			 */
 			required: { type: Boolean, reflect: true },
+			/**
+			 * Override for the time input default value. If not provided, this will default to midnight. Should be in format "hh:mm:ss".
+			 */
```

Comment:
Let's borrow the description from `d2l-input-time`:

```suggestion
			/**
			 * Set default value of time portion of the input. Valid values are times in ISO 8601 time format ("hh:mm:ss"), "startOfDay", "endOfDay".
			 * @type {'startOfDay'|'endOfDay'|string}
			 */
```